seqr/views/apis/individual_api_tests.py

```diff
@@ -463,6 +511,310 @@ def test_individuals_table_handler(self):
         response = self.client.post(save_url)
         self.assertEqual(response.status_code, 200)
 
+    @mock.patch('seqr.views.utils.permissions_utils.PM_USER_GROUP', 'project-managers')
+    @mock.patch('seqr.views.utils.pedigree_info_utils.NO_VALIDATE_MANIFEST_PROJECT_CATEGORIES')
+    @mock.patch('seqr.utils.communication_utils.EmailMultiAlternatives')
+    def test_individuals_sample_manifest_table_handler(self, mock_email, mock_no_validate_categories):
```

Comment:
this is broadly speaking all copy pasted from the other test file, just reformatted to calling the endpoint instead of the helper function directly